pages/wizard/product_selection.py

```diff
@@ -53,6 +81,30 @@ def select_products(self, products):
         if 'ocp' in products:
             self.click_openshift()
 
+    # Note: if you get 'Element is not clickable' error when using this,
+    # it's most probably because another tooltip is already covering
+    # the area of the 'i' icon you want to click.
+    # Example:
+    # First you get the text on the osp tooltip
+    # Then you want to get the rhv tooltip, but that is covered by 
+    # the osp tooltip that is still being displayed.
+    # Solution would be to move the cursor away or to click anywhere
+    # on the page to make the tooltip go away.
+    def get_i_icon_text(self, product):
```

Comment:
I would probably opt to take a dictionary approach to this where the key is the product string and the value is the function name to use.   I do something like this in the task bar code, but with library and class names.   The advantage is you replicate the same code over and over again, so if you have to change some part later you don't have to change every if clause.   At this point its not really a big deal but if the product list grows much more it would be.